cypress/support/utils.ts

```diff
@@ -24,10 +24,13 @@ export async function runTests(matchName) {
 }
 
 export function runCommand(command: string, cwd = ROOT) {
-  return execSync(command, {
+  const [cmd, ...args] = command.split(' ');
+  const fullCommand = [cmd, ...args].join(' ');
```

Comment:
in the end the command being ran is the same right? Your security scanner happens to prefer this syntax?

Reply:
@willydouhard i based this on gpt4 feedback but you're right, this doesn't seem to remove the vunerability. The scanner doesn't run every day and the last check (where I manually added this) still has it.

Any other ideas?